build/storage/Dockerfile

```diff
@@ -0,0 +1,178 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+# NOTICE: THIS FILE HAS BEEN MODIFIED BY INTEL CORPORATION UNDER COMPLIANCE
+# WITH THE APACHE 2.0 LICENSE FROM THE ORIGINAL WORK
+#
+################################################################################
+# build_base
+#
+# This image is used to provide environment for spdk build and get it in the
+# form of spdk packages
+################################################################################
+FROM fedora:33 AS base
```

Comment:
Fedora 33 is super ancient. Given that #134 is already open, I think this should be updated to use Fedora 36.

Reply:
Updated to fedora:36

Reply:
Thanks!